internal/testutils/jimmtest/mocks/jimm_ssh_mock.go

```diff
@@ -5,15 +5,16 @@ package mocks
 import (
 	"context"
 
+	gossh "golang.org/x/crypto/ssh"
+
 	"github.com/canonical/jimm/v3/internal/errors"
-	"github.com/canonical/jimm/v3/internal/jimm/ssh"
 	"github.com/canonical/jimm/v3/internal/openfga"
 )
 
 // SSHManager is an implementation of the SshManager interface.
 type SSHManager struct {
```

Comment:
Be nice to eventually just gomock interfaces for our services in packages that consume them